README.md

```diff
@@ -123,16 +124,21 @@ Install all the dependencies from your Gemfile using Bundler:
 
 	$ bundle install
 
-Run any database migrations:
+Set up your database and seed sample data (including admin users to log in!):
 
 	$ bundle exec rake db:migrate
+	$ bundle exec rake db:seed
```

Comment:
Yep this one was a bit of a fuzzy decision on my part- I've kept them separate as usually people will come back to the README after initial setup for database updates, so wanted to kill two birds and show commands for both seeding and running migrations.